src/modules/user/domain/user.service.ts

```diff
@@ -0,0 +1,38 @@
+import { ClassProvider, Inject, Injectable } from "@nestjs/common";
+import {
+	ServiceCreateUserInput,
+	ServiceUser,
+	UserServiceDefinition,
+	UserServiceName,
+} from "src/modules/user/domain/user.service.types";
+import {
+	RepoUser,
+	UserRepositoryDefinition,
+	UserRepositoryName,
+} from "src/modules/user/repository/user.repository.types";
+
+@Injectable()
```

Comment:
This @Injectable means we can inject this service anywhere throughout our app using Nest Injection, and it'll just work nicely